addons/binding/org.openhab.binding.samsungtv/README.md

```diff
@@ -14,6 +14,7 @@ Tested TV models:
 | UE46E5505 | OK      | Initial contribution is done by this model                                                                       |
 | UE46D5700 | PARTIAL | Supports at my home only commands via the fake remote, no discovery                                              |
 | UE40F6500 | OK      | All channels except `colorTemperature`, `programTitle` and `channelName` are working                             |
+| UN50J5200 | PARTIAL | Status is retrieved (`power`, `media title`). Not operatable.                                                    |
```

Comment:
I think the English word is "Not operational"

Reply:
I actually meant "operatable", in an attempt to say the openHAB doesn't seem to be able to operate the device (while getting some of the status worked). I updated the words to avoid confusion.